README.md

```diff
@@ -908,7 +908,7 @@ Community Repos:
 
 Community Repos:
 
-* [capitalone/cloud-custodian :fire::fire::fire::fire:](https://github.com/capitalone/cloud-custodian) - Rules engine for management, DSL in yaml for query, filter, and actions on resources.
+* [cloudcustodian/cloud-custodian :fire::fire::fire::fire::fire:](https://github.com/capitalone/cloud-custodian) - Rules engine for management, DSL in yaml for query, filter, and actions on resources.
```

Comment:
also update the url to point at cloudcustodian/cloud-custodian

Reply:
should also be cloud-custodian/cloud-custodian

Reply:
fixed, thanks